src/patterns/patterns.py

```diff
@@ -734,11 +734,11 @@ def get_busfactor_data(self, locc_metric='change-size-cos', metric='mul-changes-
         else:
             d = pd.DataFrame(work_df.groupby(['filepath', 'unique_author'])[locc_metric].sum())
             d["dev_knowledge"] = 0
-        d.reset_index(level=d.index.names, inplace=True)
+        d.reset_index(inplace=True)
```

Comment:
It would help to explain in the PR description why this change was necessary. But I'll ask here and approve after I learn the answer. :-)

Reply:
Sure! :-) Since the data frames don't have multi-index, I removed the level parameter which wasn't required. Tested to make sure everything works just fine.